_posts/2019-08-28-mentor-spotlight-katharine-hyatt.markdown

```diff
@@ -0,0 +1,60 @@
+
+Working on open source projects has also professionally beneficial for me. My peers and other industry professionals have come to recognize me as the "Julia person". My work has also led me to receive invitations to speak at conferences that are not just focussed on Physics and helped me explore fields broader than my immediate research interests. Having the experience of working on open source projects also makes it easier for me to find professional endeavors in the industry if I decide to explore areas outside of academia.
+
+### What about the open source community, and in particular Julia, drew you to being an active part of it? 
+As I mentioned earlier, the first language I learned was C++, and I only started using Julia in Graduate School. The selling point of Julia is that it offers the ease of use of Python with the speed of C++. What attracted me was the very welcoming and helpful community. I remember that when I first started, I didn't have a lot of git skills. One time I had to use rebase, but I had no experience in using it. That's when members from the community came forward to help me with git and also with getting my code across that production line, which was very assuring. That experience was definitely something that stuck with me and made me want to be part of this wonderful community. 
+
+### What was your first interaction/impression of open source and how has your involvement in open source evolved over time? 
+I first started contributing by writing tests and fixing edge cases in my free time, usually when waiting for simulations to finish running. My first Julia PR was on January 2nd, 2015, where I added some tests to Reduce.jl. At the time, the language was not as mature as it is today, and I occasionally encountered little bugs and missing features. 
+
+There have been some pretty noticeable changes as the Julia community has grown in the past few years. Today I hear comments like, "This stuff is so reliable" or "This never breaks", which was unheard of in the past. 
+
+I have also grown with the community, and have personally gained a lot of knowledge of experience. Today I have the 9th most commits to the language, and my experience has changed my role in the community in different ways. Now contributors sometimes ask me to review and merge PRs, which can be a little inconvenient when I have other stuff going. Now contributors sometimes ask me to review and merge PRs, which can be a little inconvenient when I have other stuff going. This is very different from my early days when I could do things at my own pace. I'm no longer also the only woman in the Julia community, and so I don't have to play the role of community manager whenever people wanted the opinion of women in Julia. 
```

Comment:
Duplicate sentence: "Now contributors sometimes ask me to review and merge PRs, which can be a little inconvenient when I have other stuff going."